recipes/jsonschema-default/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "jsonschema-default" %}
+{% set version = "1.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jsonschema-default-{{ version }}.tar.gz
+  sha256: f2ef00f3e90062bf433ee16731720844429c6b1569d3a58c9329aa9305630a87
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6,<4.0
```

Comment:
From the upstream `pyproject.toml`, it looks like the minimum python version is 3.7

Reply:
Good catch! I just rolled with what grayskull gave me and didn't check, so thanks!